api/http/indigo_service/indigo_http.py

```diff
@@ -0,0 +1,216 @@
+    response_model_exclude_unset=True,
+)
+def exact_match(request: jsonapi.MatchRequest) -> jsonapi.MatchResponse:
+    compound, *_ = service.extract_compounds(source(request))
+    target_pairs = targets(request)
+    target_compounds = service.extract_compounds(target_pairs)
+    output_format = request.data.attributes.outputFormat
+    results = []
+    for target_compound in target_compounds:
+        match = indigo().exactMatch(
+            compound, target_compound, request.data.attributes.flag
+        )
+        try:
+            results.append(
+                service.map_match_output(
+                    match, request.data.attributes.outputFormat, compound
+                )
+            )
+        except AttributeError:
+            results.append("")
+
+    return jsonapi.make_match_response(results, output_format)  # type: ignore
+
+
+@app.post(
+    f"{BASE_URL_INDIGO}/convert",
+    response_model=jsonapi.CompoundResponse,
+    response_model_exclude_unset=True,
+)
+def convert(
+    request: jsonapi.CompoundConvertRequest,
+) -> jsonapi.CompoundResponse:
+    compound, *_ = service.extract_compounds(compounds(request))
+    return jsonapi.make_compound_response(
+        *service.to_string(compound, request.data.attributes.outputFormat)
+    )
+
+
+@app.post(
+    f"{BASE_URL_INDIGO}/validate",
+    response_model=jsonapi.ValidationResponse,
+    response_model_exclude_unset=True,
+)
+def validate(request: jsonapi.ValidationRequest) -> jsonapi.ValidationResponse:
+    compound, *_ = service.extract_compounds(compounds(request))
+    validations = request.data.attributes.validations
+    results = {}
+    for validation in validations:
+        results[validation] = service.validate(compound, validation)
+    return jsonapi.make_validation_response(results)
+
+
+@app.post(
+    f"{BASE_URL_INDIGO}/descriptors",
+    response_model=jsonapi.DescriptorResponse,
+    response_model_exclude_unset=True,
+)
+def descriptors(
+    request: jsonapi.DescriptorRequest,
+) -> jsonapi.DescriptorResponse:
+    compound, *_ = service.extract_compounds(compounds(request))
+    properties = request.data.attributes.descriptors
+    results = {}
+    for property_ in properties:
+        results[property_] = service.get_descriptor(compound, property_)
+    return jsonapi.make_descriptor_response(results)
+
+
+@app.post(
+    f"{BASE_URL_INDIGO}/commonBits",
+    response_model=jsonapi.CommonBitsResponse,
+    response_model_exclude_unset=True,
+)
+def common_bits(
+    request: jsonapi.CommonBitsRequest,
+) -> jsonapi.CommonBitsResponse:
+    compound, *_ = service.extract_compounds(source(request))
+    target_compounds = service.extract_compounds(targets(request))
+    source_fp = compound.fingerprint("sim")
+    result = []
+    for target in target_compounds:
+        bits = indigo().commonBits(source_fp, target.fingerprint("sim"))
+        result.append(bits)
+    return jsonapi.make_common_bits_response(result)
+
+
```

Comment:
Is it complicated to add missing methods for required by Ketcher? It seems like the following are required:
* `aromatize`
* `dearomatize`
* `layout`
* `clean2d`
Also `render` is quite important, but it could be done later.

Reply:
Added `aromatize`, `dearomatize`, `clean2d` to /indigo/convert endpoint. 

`layout` and `render` needs clarification, let's move it to feature request?

Reply:
`layout` added. `render` will be added in the next release